### ISSUE TITLE
ctags-win32/releases seem to have dropped docs, man dirs & content

### ISSUE THREAD

**Issue by fwmechanic** [universal-ctags/ctags#1962]
Between the [2018-12-14](https://github.com/universal-ctags/ctags-win32/releases/tag/2018-12-14%2F64f7d619) and [2018-12-16](https://github.com/universal-ctags/ctags-win32/releases/tag/2018-12-16%2F9d32534c) ctags-win32 releases, the contents of doc and man dirs seems to have been dropped from the release package; the `ctags-*-x64.zip` file has decreased in size from 2.05MB to 1.65MB across this boundary (and remains true [as of most recent release as of this writing](https://github.com/universal-ctags/ctags-win32/releases/tag/2018-12-26%2F5728abe4)).  See attached `7z t` dumps from the two above-mentioned files and the diff thereof:

[ctags-2018-12-14_64f7d619-x64.txt](https://github.com/universal-ctags/ctags/files/2710959/ctags-2018-12-14_64f7d619-x64.txt)  or

[ctags-2018-12-16_9d32534c-x64.txt](https://github.com/universal-ctags/ctags/files/2710960/ctags-2018-12-16_9d32534c-x64.txt)

[diff.txt](https://github.com/universal-ctags/ctags/files/2710961/diff.txt)

A quick glance at the commits in between does not give me a clue as to why this would have happened; it does not seem to have been intentional...

**Comment by k-takata** [comment]
https://ci.appveyor.com/project/universalctags/ctags-win32/builds/21229595/job/10t7guonmj391722?fullLog=true#L1147

```
sphinx-build -b html -d _build/doctrees   . _build/html
Traceback (most recent call last):
  File "C:/msys64/mingw64/lib/python3.7/site-packages\pkg_resources\__init__.py", line 581, in _build_master
    ws.require(__requires__)
  File "C:/msys64/mingw64/lib/python3.7/site-packages\pkg_resources\__init__.py", line 898, in require
    needed = self.resolve(parse_requirements(requirements))
  File "C:/msys64/mingw64/lib/python3.7/site-packages\pkg_resources\__init__.py", line 789, in resolve
    raise VersionConflict(dist, req).with_context(dependent_req)
pkg_resources.ContextualVersionConflict: (idna 2.8 (c:/msys64/mingw64/lib/python3.7/site-packages), Requirement.parse('idna<2.8,>=2.5'), {'requests'})
```

It seems that MSYS2 packaging has a problem. There is a version conflict between "requests" and "idna" (which are used by sphinx).
I created an issue: Alexpux/MINGW-packages#4802

**Comment by masatake** [comment]
It seems that there is mismatch between the version of python installed to appveyor and the version of python sphinx expects.
https://ci.appveyor.com/project/universalctags/ctags-win32/builds/21135185/job/d8wynf0iia6wtaef?fullLog=true

**Comment by masatake** [comment]
Oh, @k-takata has already started working on this item. Thank you.

**Comment by fwmechanic** [comment]
I confirm that this issue is fixed as of [2018-12-29/18bf3a48](https://github.com/universal-ctags/ctags-win32/releases/tag/2018-12-29%2F18bf3a48).
